code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -3,6 +3,8 @@
 
 #define FAILED_TO_ADD 1
 
+/obj/item/borg/upgrade/var/vtec_bonus = 0.5
```

Comment:
Not quite sure how to declare vars confined to a single dm honestly, and managing the bonus in both the reset and vtec module seemed bad.